solidity/package.json

```diff
@@ -1,33 +1,42 @@
 {
-    "name": "@keep-network/tbtc-v2",
-    "license": "MIT",
-    "dependencies": {
-      "@openzeppelin/contracts": "^4.1.0"
-    },
-    "devDependencies": {
-      "@nomiclabs/hardhat-ethers": "^2.0.2",
-      "@nomiclabs/hardhat-waffle": "^2.0.1",
-      "chai": "^4.3.4",
-      "eslint": "^7.27.0",
-      "eslint-config-keep": "github:keep-network/eslint-config-keep#0.3.0",
-      "ethereum-waffle": "^3.3.0",
-      "ethers": "^5.3.0",
-      "hardhat": "^2.3.0",
-      "hardhat-gas-reporter": "^1.0.4",
-      "prettier": "^2.3.0",
-      "prettier-plugin-solidity": "^1.0.0-beta.11 ",
-      "solhint": "^3.3.6",
-      "solhint-config-keep": "github:keep-network/solhint-config-keep",
-      "typescript": "^4.3.2"
-    },
-    "scripts": {
-      "build": "hardhat compile",
-      "lint": "npm run lint:js && npm run lint:sol",
-      "lint:fix:js": "eslint . --fix",
-      "lint:fix:sol": "solhint 'contracts/**/*.sol' --fix && prettier --write '**/*.sol'",
-      "lint:js": "eslint . ",
-      "lint:sol": "solhint 'contracts/**/*.sol' && prettier --check '**/*.sol'",
-      "test": "hardhat test"
-    }
+  "name": "@keep-network/tbtc-v2",
+  "license": "MIT",
+  "scripts": {
```

Comment:
I believe we'll need to add "format", "format:fix" and change other commands accordingly just like we have in `/yearn` dir